source/includes/_050-materials-webhook.md.erb

```diff
@@ -0,0 +1,6 @@
+# Notify Materials
```

Comment:
Can we say webhook in title?

Reply:
Not sure. We haven't used it in any doc.

Reply:
Webhook and notify are two different APIs

### Notify API
https://github.com/gocd/gocd/blob/5559396a8164083734c05b0d211d461496d73d61/spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java#L185-L191

### Webhook API
https://github.com/gocd/gocd/blob/5559396a8164083734c05b0d211d461496d73d61/spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java#L867-L873

Reply:
So the webhook API you are mentioning isn't documented, isn't it

Reply:
It is documented [here](https://api.gocd.org/20.1.0/#hosting-your-repository-on-bitbucket)

<img width="726" alt="Screenshot 2020-01-30 at 12 07 32 PM" src="https://user-images.githubusercontent.com/7871209/73425964-2fb89e00-4359-11ea-918d-569ac19c06b0.png">

Reply:
Sorry missed it.

Reply:
Added another header `Webhook` which contains the above mentioned APIs.